src/styles/_catalog.scss

```
@@ -1,9 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  grid-template-columns: repeat(4, auto);
+  justify-content: center;
+  align-items: center;
```

```suggestion
  align-items: center;
```
do you really need this? 🤔

---

Ohh! really... i dont, thank you for your advice!